# Message header
std_msgs/Header header

# Robot pose
geometry_msgs/Pose2D pose